app/src/main/java/com/kickstarter/ui/activities/MessagesActivity.java

```diff
@@ -0,0 +1,116 @@
+package com.kickstarter.ui.activities;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Pair;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import com.jakewharton.rxbinding.widget.RxTextView;
+import com.kickstarter.R;
+import com.kickstarter.libs.BaseActivity;
+import com.kickstarter.libs.KSCurrency;
+import com.kickstarter.libs.KSString;
+import com.kickstarter.libs.qualifiers.RequiresActivityViewModel;
+import com.kickstarter.libs.utils.DateTimeUtils;
+import com.kickstarter.libs.utils.ViewUtils;
+import com.kickstarter.models.Backing;
+import com.kickstarter.models.Project;
+import com.kickstarter.ui.adapters.MessagesAdapter;
+import com.kickstarter.viewmodels.MessagesViewModel;
+
+import butterknife.Bind;
+import butterknife.BindString;
+import butterknife.ButterKnife;
+
+import static com.kickstarter.libs.rx.transformers.Transformers.observeForUI;
+
+@RequiresActivityViewModel(MessagesViewModel.ViewModel.class)
+public final class MessagesActivity extends BaseActivity<MessagesViewModel.ViewModel> {
+  private KSCurrency ksCurrency;
+  private KSString ksString;
+  private MessagesAdapter adapter;
+
+  protected @Bind(R.id.backing_amount_text_view) TextView backingAmountTextViewText;
+  protected @Bind(R.id.backing_info_view) View backingInfoView;
+  protected @Bind(R.id.messages_creator_name_text_view) TextView creatorNameTextView;
```

Comment:
think we need to rename all of the creator specific stuff to just "participant" in order to leave room for future feature development